### DISCUSSION TITLE
typescript could not found any module

### DISCUSSION THREAD

**Discussion by siegaii** [pnpm/pnpm#4783] [Q&A]
I'm using pnpm to install packages for my project, and after installing packages, I saw that there is a .pnpm folder in the node_module directory, all packages are placed into the .pnpm folder, this cause a problem that ts could not found any module. what should I do to solve it?

![image](https://user-images.githubusercontent.com/40651327/169808900-4ccbbd2e-7b37-4f9a-ba9e-28651ae4b4d9.png)

**Comment by woss** [comment]
have you tried to do this? 
![image](https://user-images.githubusercontent.com/80440/170080603-4f50df37-3c92-4b06-8584-76c8fd35d96a.png)
